admin/edit_project.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+require 'functions.php';
+
+$db = connectDatabase();
+
+$stmt = $db->prepare("SELECT `id`,`title`,`mini_description`,`background_image`,`project_url` FROM `projects` WHERE `id` = :id ;");
+$stmt->bindParam(':id', $_GET["id"]);
+$stmt->execute();
+$projects_data = $stmt->fetchAll();
```

Comment:
if you are just fetching one record you can use fetch instead of fetchAll then you won't have to do $projects_data[0]["mini_description"] you can just do $projects_data["mini_description"]

Reply:
changed - thanks